batch/pixetl.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM globalforestwatch/pixetl:v1.4.0
+FROM globalforestwatch/pixetl:v1.4.1
```

Comment:
v1.4.2 includes your latest `pixetl_prep` changes. Not sure if they were breaking changes and you would need to adopt any of the code. If not, please update